docs/docs/webchat/webchat-storage.md

```diff
@@ -0,0 +1,49 @@
+---
+id: webchat-storage
+title: Storage
+---
+
+## Local and Session Storage
+
+By default, Botonic stores its state in the [**localStorage**](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) browser where the data doesn't expire.
+
+However, for security and privacy reasons, you may have to avoid storing bot data.
+
+That's why Botonic also allows you to store on the [**sessionStorage**](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) where everything is cleared once the webchat window is closed or reloaded.
+
+If you don't want to store anything in the browser, the **`null`** variable is recommended.
```

Comment:
"If you don't want to store anything in the browser, you must set `storage` to  **`null`**."

"Not recommended, is what you need to do.